src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -526,6 +526,7 @@ public enum SyntaxKind : ushort
         Utf8StringLiteralToken = 8520,
         Utf8SingleLineRawStringLiteralToken = 8521,
         Utf8MultiLineRawStringLiteralToken = 8522,
+        BadRazorContentToken = 8523,
```

Comment:
>BadRazorContentToken [](http://example.com/codeflow?start=8&length=20)

The name `BadRazorContentToken` sounds, to me, like content that is unexpected in Razor. But it seems like the token is valid in Razor. Would it make sense to simply call this `RazorContentToken`?

Reply:
I'm ok with that rename.